packages/upload-api/test/helpers/blob.js

```diff
@@ -54,6 +60,18 @@ export function parseBlobAddReceiptNext(receipt) {
     throw new Error('mandatory effects not received')
   }
 
+  let acceptSite = {}
+  if (acceptReceipt?.out.ok?.site) {
+    const blocks = new Map(
+      [...receipt.iterateIPLDBlocks()].map((block) => [`${block.cid}`, block])
+    )
+
+    acceptSite.site = Delegation.view({
+      root: /** @type {API.UCANLink} */ (acceptReceipt.out.ok.site),
+      blocks,
+    })
+  }
```

Comment:
@joaosa this is exact code (which we discussed) that resolves site commitment from the receipt